environments/icds-cas/known_hosts

```diff
@@ -630,4 +639,10 @@
 100.71.188.75 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDabZXv/2VeZu6b2CQYI7xvzisSRcgMRQ+7neY0kkwX+xI/EiM/LNBXUm5heLSq68OJpUIJ/k6vQV2AtafblDWQMCNScQ0LhaDaMhwqVhLNvI4kozm/LSgy4mSgV8Z7PXa7FMhavmexhGlu4X2MbMu7Me88KwPEwEwxhEDnjsoD/AWTqRzmyKsNbTtqxwF9q/uhD0PFtObTb093qghY4ST07ju+8nn70fGQJ4Fw6MeqrQjt69n4wkA5vnVI5BNhQofOV/CNqBM7oo4+tO/lwHn/UQcPXThgi9rjTo6d6q0hm7oRFxDIWBxHzp/XPByTZMuj/GLrfUBp8Rvar8j91o21
 100.71.188.78 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBL++uNtc5WXpSb1t3ooZmVJxYRilzqJc9KspY2rnuM0KmQ3uTVC3XVmeEUq4qn7hTZInOJpTEZneWYTnU89pgrM=
 100.71.188.78 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIETMDvQ592lfJ+vDeEGaZFgyGpWzPVB+hgzuQq/as6ZN
-100.71.188.78 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDabZXv/2VeZu6b2CQYI7xvzisSRcgMRQ+7neY0kkwX+xI/EiM/LNBXUm5heLSq68OJpUIJ/k6vQV2AtafblDWQMCNScQ0LhaDaMhwqVhLNvI4kozm/LSgy4mSgV8Z7PXa7FMhavmexhGlu4X2MbMu7Me88KwPEwEwxhEDnjsoD/AWTqRzmyKsNbTtqxwF9q/uhD0PFtObTb093qghY4ST07ju+8nn70fGQJ4Fw6MeqrQjt69n4wkA5vnVI5BNhQofOV/CNqBM7oo4+tO/lwHn/UQcPXThgi9rjTo6d6q0hm7oRFxDIWBxHzp/XPByTZMuj/GLrfUBp8Rvar8j91o21
\ No newline at end of file
+100.71.188.78 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDabZXv/2VeZu6b2CQYI7xvzisSRcgMRQ+7neY0kkwX+xI/EiM/LNBXUm5heLSq68OJpUIJ/k6vQV2AtafblDWQMCNScQ0LhaDaMhwqVhLNvI4kozm/LSgy4mSgV8Z7PXa7FMhavmexhGlu4X2MbMu7Me88KwPEwEwxhEDnjsoD/AWTqRzmyKsNbTtqxwF9q/uhD0PFtObTb093qghY4ST07ju+8nn70fGQJ4Fw6MeqrQjt69n4wkA5vnVI5BNhQofOV/CNqBM7oo4+tO/lwHn/UQcPXThgi9rjTo6d6q0hm7oRFxDIWBxHzp/XPByTZMuj/GLrfUBp8Rvar8j91o21
+100.71.189.15 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBL++uNtc5WXpSb1t3ooZmVJxYRilzqJc9KspY2rnuM0KmQ3uTVC3XVmeEUq4qn7hTZInOJpTEZneWYTnU89pgrM=
+100.71.189.15 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIETMDvQ592lfJ+vDeEGaZFgyGpWzPVB+hgzuQq/as6ZN
+100.71.189.15 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDabZXv/2VeZu6b2CQYI7xvzisSRcgMRQ+7neY0kkwX+xI/EiM/LNBXUm5heLSq68OJpUIJ/k6vQV2AtafblDWQMCNScQ0LhaDaMhwqVhLNvI4kozm/LSgy4mSgV8Z7PXa7FMhavmexhGlu4X2MbMu7Me88KwPEwEwxhEDnjsoD/AWTqRzmyKsNbTtqxwF9q/uhD0PFtObTb093qghY4ST07ju+8nn70fGQJ4Fw6MeqrQjt69n4wkA5vnVI5BNhQofOV/CNqBM7oo4+tO/lwHn/UQcPXThgi9rjTo6d6q0hm7oRFxDIWBxHzp/XPByTZMuj/GLrfUBp8Rvar8j91o21
+;; ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBL++uNtc5WXpSb1t3ooZmVJxYRilzqJc9KspY2rnuM0KmQ3uTVC3XVmeEUq4qn7hTZInOJpTEZneWYTnU89pgrM=
+;; ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIETMDvQ592lfJ+vDeEGaZFgyGpWzPVB+hgzuQq/as6ZN
+;; ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDabZXv/2VeZu6b2CQYI7xvzisSRcgMRQ+7neY0kkwX+xI/EiM/LNBXUm5heLSq68OJpUIJ/k6vQV2AtafblDWQMCNScQ0LhaDaMhwqVhLNvI4kozm/LSgy4mSgV8Z7PXa7FMhavmexhGlu4X2MbMu7Me88KwPEwEwxhEDnjsoD/AWTqRzmyKsNbTtqxwF9q/uhD0PFtObTb093qghY4ST07ju+8nn70fGQJ4Fw6MeqrQjt69n4wkA5vnVI5BNhQofOV/CNqBM7oo4+tO/lwHn/UQcPXThgi9rjTo6d6q0hm7oRFxDIWBxHzp/XPByTZMuj/GLrfUBp8Rvar8j91o21
```

Comment:
I think these last 3 lines should be removed

Reply:
is it just me or do a lot of these hosts have exactly the same keys?

Reply:
> I think these last 3 lines should be removed

These lines were added  when ' cchq icds update-local-known-hosts' was run .
Let me remove them and see how the travis goes .

Reply:
> is it just me or do a lot of these hosts have exactly the same keys?

I also see three different keys one for "ssh-rsa" , one for "ssh-ed25519" and one for "ecdsa-sha2-nistp256"  and this is common for all the hosts.

Reply:
> > I think these last 3 lines should be removed
> 
> These lines were added when ' cchq icds update-local-known-hosts' was run .
> Let me remove them and see how the travis goes .

Removed additonal 3 lines under commit 
https://github.com/dimagi/commcare-cloud/pull/3761/commits/381743452aa754d5421e777e1fbef713ee9abc24

Reply:
What I mean is that if you search for `ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDabZXv/2VeZu6b2CQYI7xvzisSRcgMRQ+7neY0kkwX+xI/EiM/LNBXUm5heLSq68OJpUIJ/k6vQV2AtafblDWQMCNScQ0LhaDaMhwqVhLNvI4kozm/LSgy4mSgV8Z7PXa7FMhavmexhGlu4X2MbMu7Me88KwPEwEwxhEDnjsoD/AWTqRzmyKsNbTtqxwF9q/uhD0PFtObTb093qghY4ST07ju+8nn70fGQJ4Fw6MeqrQjt69n4wkA5vnVI5BNhQofOV/CNqBM7oo4+tO/lwHn/UQcPXThgi9rjTo6d6q0hm7oRFxDIWBxHzp/XPByTZMuj/GLrfUBp8Rvar8j91o21` in the file you will see that many hosts have the same key. Also the same for the other key types. 

Could this be related to how TCL are creating the VMs? From my understanding they should all have unique keys.

Reply:
Discussed this with RohitK and he has figured out that the it is hardcode in the Ubuntu template that TCL uses to build the VMs

root@MUMGCCWCDPRDWV001:/etc/ssh# cat ssh_host_rsa_key.pub 
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDabZXv/2VeZu6b2CQYI7xvzisSRcgMRQ+7neY0kkwX+xI/EiM/LNBXUm5heLSq68OJpUIJ/k6vQV2AtafblDWQMCNScQ0LhaDaMhwqVhLNvI4kozm/LSgy4mSgV8Z7PXa7FMhavmexhGlu4X2MbMu7Me88KwPEwEwxhEDnjsoD/AWTqRzmyKsNbTtqxwF9q/uhD0PFtObTb093qghY4ST07ju+8nn70fGQJ4Fw6MeqrQjt69n4wkA5vnVI5BNhQofOV/CNqBM7oo4+tO/lwHn/UQcPXThgi9rjTo6d6q0hm7oRFxDIWBxHzp/XPByTZMuj/GLrfUBp8Rvar8j91o21 root@UBUNTU18042Template

Reply:
hmm, perhaps we should regenerate the keys in future?

Reply:
I agree. I will make a plan for this.